shared/templates/accounts_password/ansible.template

```diff
@@ -5,6 +5,30 @@
 # disruption = low
 - (xccdf-var var_password_pam_{{{ VARIABLE }}})
 
+{{% if product == "ol8" %}}
+- name: Find pwquality.conf.d files
```

Comment:
It is good to include `{{{ rule_title }}}` as prefix in task names. Could you include, please?
https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible